.github/workflows/release-please.yml

```diff
@@ -28,17 +28,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node 12.X
+      - name: Use Node 16.X
         uses: actions/setup-node@v2
         with:
-          node-version: 12.x
+          node-version: 16.x
           registry-url: https://registry.npmjs.org/
```

Comment:
No need for cache here?

Reply:
it hasn't changed from before.

iirc i didn't use cache because we should take fresh deps for release